### ISSUE TITLE
Msftidy on towelroot

### ISSUE THREAD

**Issue by todb-r7** [rapid7/metasploit-framework#4747]
````
modules/exploits/android/local/futex_requeue.rb - [ERROR] Exploit is missing a disclosure date
modules/exploits/android/local/futex_requeue.rb - [WARNING] Suspect capitalization in module title: 'futex'
modules/exploits/android/local/futex_requeue.rb - [WARNING] Suspect capitalization in module title: 'requeue'
modules/exploits/android/local/futex_requeue.rb - [WARNING] Suspect capitalization in module title: 'kernel'
modules/exploits/android/local/futex_requeue.rb - [WARNING] Suspect capitalization in module title: 'exploit'
modules/exploits/android/local/futex_requeue.rb:22 - [WARNING] Spaces at EOL
modules/exploits/android/local/futex_requeue.rb:26 - [WARNING] Spaces at EOL
modules/exploits/android/local/futex_requeue.rb:77 - [WARNING] Spaces at EOL
modules/exploits/android/local/futex_requeue.rb - [WARNING] Module contains old license comment, use tools/dev/resplat.rb <filename>.
````

Naughty. @bcook-r7 @timwr please to be using the msftidy pre-commit hooks :) tools/dev/pre-commit-hook.rb

**Comment by bcook-r7** [comment]
I'll fix it - it's the only way I'll learn :P

**Comment by wvu-r7** [comment]
For the "suspect capitalization" warnings, you can add "futex" and "requeue" to the whitelist in ```msftidy```.

**Comment by wvu-r7** [comment]
Actually, it should be "futex_requeue" in the title. So add that to the whitelist. :)

**Comment by wvu-r7** [comment]
Fixed in #4748.
